### PR TITLE
Mention `//` (double slash) operator

### PR DESCRIPTION
**Pull request by cweiske**
- if: https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/Functions/If.html
- Data getText: https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/Functions/Data.html#data-gettext

### CODE REVIEW

File: Documentation/Functions/If.rst [hunk 1]
```diff
@@ -20,6 +20,9 @@ There is no else property available. The "else" branch of an "if" statement is a
 missing feature. You can implement a workaround by a logic based on the
 :ref:`stdwrap-override-conditions`.
 
+Simple "if empty use different value" conditions can be built with the
+:ref:`TypoScript // (double slash) <data-type-gettext>` fallback operator.
```
Comment (brotkrueml) [inline Documentation/Functions/If.rst:24]: I suggest to add an anchor to the section and use it here, so people don't have to search for the section. What do you think?

File: Documentation/Functions/If.rst [hunk 2]
```diff
@@ -20,6 +20,9 @@ There is no else property available. The "else" branch of an "if" statement is a
 missing feature. You can implement a workaround by a logic based on the
 :ref:`stdwrap-override-conditions`.
 
+Simple "if empty use different value" conditions can be built with the
```
Comment (brotkrueml) [inline Documentation/Functions/If.rst:23]:
```suggestion
Simple "if empty use different value" conditions can be build with the
```

**Review by brotkrueml — COMMENTED** [review]
Thanks for your improvement :+1: I have two remarks.

File: Documentation/Functions/If.rst (hunk 2 above)
Comment (cweiske) [reply Documentation/Functions/If.rst:23]: "can be built" is correct english.
https://textranch.com/c/can-be-built-or-can-be-build/

**Comment by froemken** [comment]
Hello @cweiske 

yeah...the double slash operator is a pretty awesome feature, but please have a look here:

https://github.com/TYPO3/typo3/blob/v13.2.1/typo3/sysext/frontend/Classes/ContentObject/ContentObjectRenderer.php#L3762

The operator is only available for TS property "field". So, currently I can not merge your patch as long as that patch will add it directly into the global gettext-part.

Stefan

**Comment by cweiske** [comment]
I clarified it so that it is clear that this option is for record data only.

**Review by linawolf — APPROVED** [review]
Cool Feature! Didnt even know about it!

**Review by linawolf — DISMISSED** [review]
Woops, sorry, no I think @froemken s comment ist still valid here. It is not a property of gettext but of field...

**Comment by linawolf** [comment]
Actually looking at https://github.com/TYPO3/typo3/blob/v13.2.1/typo3/sysext/frontend/Classes/ContentObject/ContentObjectRenderer.php#L3791 I would say it is a feature of both field and gettext, what would you say @froemken ?

**Review by froemken — APPROVED** [review]

**Comment by froemken** [comment]
@linawolf Right. That feature is there for gettext since ages. Checked that until 9 years ago...and it is still older than 9 years.

**Pull request merged**